service/src/main/java/bio/terra/workspace/service/workspace/model/WsmObjectType.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.service.workspace.model;
+
+public enum WsmObjectType {
```

Comment:
This looks similar to ActivityLogChangedTarget. Can they be combined? Perhaps an enum with methods/properties.

Reply:
good point :) will remove this class